roles/av/tasks/main.yml

```diff
@@ -20,6 +20,14 @@
     state: present
     create: yes
 
+- name: ensure user is in audio group
```

Comment:
It's weird that this worked for me without this. I see code in my config to add the `mpd` user to the `audio` group, but not the user running ansible.